pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/model/elastic/MedicalRecordFinalObjectTransformerTest.java

```diff
@@ -0,0 +1,66 @@
+
+package org.broadinstitute.dsm.model.elastic;
+
+import java.util.List;
+import java.util.Map;
+
+import org.broadinstitute.dsm.TestHelper;
+import org.broadinstitute.dsm.db.MedicalRecordFinalDto;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class MedicalRecordFinalObjectTransformerTest {
+
+    @BeforeClass
+    public static void setUp() {
+        TestHelper.setupDB();
```

Comment:
Since it has the connection to database is no longer unit test and is way more better to move it to `integration` directory.